src/ui/component/fileList/view.jsx

```diff
@@ -11,19 +12,31 @@ const SORT_OLD = 'old';
 
 type Props = {
   uris: Array<string>,
-  header: React.Node,
-  headerAltControls: React.Node,
-  injectedItem?: React.Node,
+  header: Node,
+  headerAltControls: Node,
+  injectedItem?: Node,
   loading: boolean,
   noHeader?: boolean,
   slim?: string,
   empty?: string,
+  meta?: Node,
   // If using the default header, this is a unique ID needed to persist the state of the filter setting
   persistedStorageKey?: string,
 };
 
 export default function FileList(props: Props) {
-  const { uris, header, headerAltControls, injectedItem, loading, persistedStorageKey, noHeader, slim, empty } = props;
+  const {
+    uris,
+    header,
```

Comment:
`header` and `noHeader` could be a single param, unless it's considered a bad practice to say a type can be a `Node` or  `boolean` (true = default, false = none, node = inject)

Reply:
No I think that makes more sense. Will update.